packages/node-sdk/README.md

```diff
@@ -26,28 +26,25 @@ in **Bucket.co**.
 > information that is often sensitive and thus should not be used in
 > client-side applications.
 
+Create a `bucket.ts` file containing the following and set up the
+BUCKET_SECRET_KEY environment variable:
+
 ```ts
 import { BucketClient } from "@bucketco/node-sdk";
 
 // Create a new instance of the client with the secret key. Additional options
-// are available, such as supplying a logger, fallback features and
-// other custom properties.
-//
-// Fallback features are used in the situation when the server-side
-// features are not yet loaded or there are issues retrieving them.
-// See "Initialization Options" below for more information.
+// are available, such as supplying a logger and other custom properties.
 //
 // We recommend that only one global instance of `client` should be created
 // to avoid multiple round-trips to our servers.
-const client = new BucketClient({
-  secretKey: "sec_prod_xxxxxxxxxxxxxxxxxxxxx",
-  fallbackFeatures: ["huddle", "voice-huddle"],
-});
+export const bucketClient = new BucketClient();
 
 // Initialize the client and begin fetching feature targeting definitions.
 // You must call this method prior to any calls to `getFeatures()`,
 // otherwise an empty object will be returned.
-await client.initialize();
+client.initialize().then({
+  console.log("Bucket initialized!")
+})
```

Comment:
what's this `client` ? did you mean `bucketClient` ?

Reply:
right, doh. Good catch